docs/src/heaps.md

```diff
@@ -7,7 +7,7 @@ All heaps in this package are derived from `AbstractHeap`, and provide
 the following interface:
 
 ```julia
-# Let h be a heap, i be a handle, and v be a value.
+# Let h be a heap, i be a handle, id be a node_id, and v be a value.
```

Comment:
I believe that `i` and `id` actually refer to the same thing here and below.

Reply:
No, actually `i` is the index of the element in heap, and `id` is the actual node index. 
It's like `node_map[i] == nd_id` is `true`.

Reply:
Then is the documentation below incorrect?  It says

```julia
delete!(h, id)               # deletes the node present at node_index id from the heap
```

Reply:
Looking at the code closer, the `handle` of a node is always set to its index within the heap.  Its purpose seems to be to have a convenient way for a node to know its index.

Reply:
Actually, I had added that `delete!(h, id)` documentation, that's why it mentions `id`. 
The `update!(h, i , v)` has this line `nd_id = nodemap[i]`

Reply:
I checked this also. After we `pop!` from the heap, that particular `nodemap[i]` is set to 0.

Reply:
So @kmsquire this is fine then?